core/src/main/java/com/orange/ouds/core/component/OudsCheckbox.kt

```diff
@@ -186,23 +185,21 @@ private fun OudsCheckbox(
             modifier = Modifier
                 .size(checkboxTokens.sizeIndicator.value)
                 .selectorBorder(state = state, selected = selected, error = error, shape = shape)
-                .background(color = selectorBackgroundColor(state = state, selected = selected, error = error)),
-            contentAlignment = Alignment.Center
-        ) {
-            val selectorResource = when (value) {
-                ToggleableState.On -> R.drawable.ic_tick
-                ToggleableState.Off -> null
-                ToggleableState.Indeterminate -> R.drawable.ic_less
-            }
+        )
 
-            selectorResource?.let { resource ->
-                Icon(
-                    modifier = Modifier.fillMaxSize(),
-                    painter = painterResource(resource),
-                    tint = tickColor(state = state, error = error),
-                    contentDescription = null
-                )
-            }
+        val selectorResource = when (value) {
+            ToggleableState.On -> R.drawable.ic_tick
+            ToggleableState.Off -> null
+            ToggleableState.Indeterminate -> R.drawable.ic_less
+        }
+
+        selectorResource?.let { resource ->
+            Icon(
+                modifier = Modifier.size(checkboxTokens.sizeIndicator.value),
```

Comment:
If I'm not wrong, instead of replacing `fillMaxSize` with `size(checkboxTokens.sizeIndicator.value)` and removing the `Icon` from the `Box` content, we can simply remove `padding(all = selectorBorderWidth)` line 228.